src/PyListProxyHandler.cc

```diff
@@ -418,7 +417,7 @@ static bool array_fill(JSContext *cx, unsigned argc, JS::Value *vp) {
   }
 
   JS::RootedValue fillValue(cx, args[0].get());
-  PyObject *fillValueItem = pyTypeFactory(cx, fillValue)->getPyObject();
+  PyObject *fillValueItem = pyTypeFactory(cx, fillValue);
```

Comment:
`fillValueItem` needs to be `Py_DECREF`'d before the end of this function call

Reply:
The call to PyList_SetItem transfers ownership of the reference, even if it fails
https://docs.python.org/3/extending/extending.html#ownership-rules